src/H5FDsubfiling.c

```diff
@@ -433,18 +434,50 @@ H5FD__copy_plist(hid_t fapl_id, hid_t *id_out_ptr)
 herr_t
 H5Pset_fapl_subfiling(hid_t fapl_id, H5FD_subfiling_config_t *fa)
 {
-    H5P_genplist_t *plist     = NULL; /* Property list pointer */
-    herr_t          ret_value = FAIL;
+    H5P_genplist_t *  plist      = NULL; /* Property list pointer */
+    hid_t             ioc_fapl   = H5I_INVALID_HID;
+    H5FD_ioc_config_t ioc_config = {
+        0,
+    };
+    H5FD_subfiling_config_t subfiling_conf = {
+        0,
+    };
```

Comment:
Any particular reason for these initializers?  If I read the code correctly, you are not using these  structures, or are initializing them elsewhere.

Reply:
No reason, I can remove them.